### PR TITLE
[13.0][ADD] sale_stock_available_to_promise_release_dropshipping

### PR DESCRIPTION
**Pull request by mmequignon**
Glue module between `sale_stock_available_to_promise_release` and `stock_dropshipping`.

Based on:
- https://github.com/OCA/wms/pull/243, to be merged first

### CODE REVIEW

**Review by sebalix — APPROVED** [review]

**Review by TDu — APPROVED** [review]

**Comment by mmequignon** [comment]
@simahawk this one is ready to be merged

File: sale_stock_available_to_promise_release_dropshipping/models/sale_order_line.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
```
Comment (simahawk) [inline sale_stock_available_to_promise_release_dropshipping/models/sale_order_line.py]: mixed licenses

Comment (simahawk) [inline sale_stock_available_to_promise_release_dropshipping/models/sale_order_line.py]:
```suggestion
# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)
```

**Review by simahawk — CHANGES_REQUESTED** [review]

**Review by grindtildeath — APPROVED** [review]

File: sale_stock_available_to_promise_release_dropshipping/models/sale_order_line.py (hunk 1 above)
Comment (sebalix) [reply sale_stock_available_to_promise_release_dropshipping/models/sale_order_line.py]: Done

**Pull request merged**